src/SamplesApp/UITests.Shared/Windows_Security/Authentication_Web/AuthenticationBroker_Demo.xaml.cs

```diff
@@ -33,7 +33,7 @@ private async void PrepareClient()
 				// Create options for endpoint discovery
 				var options = new OidcClientOptions()
 				{
-					Authority = "https://demo.identityserver.io",
+					Authority = "https://demo.duendesoftware.com/",
```

Comment:
Why this one ? (how did you choose it and what should be done next time this new one does not work)

Reply:
IdentityServer from what I recall is now a commercial offering which is offered by DuendeSoftware. This demo endpoint is a service they offer: https://docs.duendesoftware.com/identityserver/v6/overview/demo_server/
If this demo server is decommissioned we may need to look at standing up our own instance for dev/testing